packages/contracts/src/dollar/core/UbiquityCreditToken.sol

```diff
@@ -31,18 +41,22 @@ contract UbiquityCreditToken is ERC20Ubiquity {
         _;
     }
 
-    /// @notice raise capital in form of Ubiquity Credit Token (only redeemable when Ubiquity Dollar > 1$)
-    /// @param amount the amount to be minted
-    /// @dev you should be minter to call that function
+    /**
+     * @notice Raises capital in the form of Ubiquity Credit Token (only redeemable when Ubiquity Dollar > 1$)
+     * @param amount Amount to be minted
+     * @dev You should be minter to call this function
```

Comment:
```suggestion
     * @dev MINTER access control role is required to call this function
```

I'm not sure if that's technically correct to say, but it might be useful to explain that we have recognized access control roles (e.g. MINTER, BURNER) and the name of the access control contract that the dev should be aware of.